addon/templates/components/sessions-grid.hbs

```diff
@@ -30,65 +27,85 @@
       <span class="session-grid-first-offering">
         {{#if (await obj.session.firstOfferingDate)}}
           {{#if obj.isIlm}}
-            <strong>{{t "general.ilm"}}: {{t "general.dueBy"}}</strong> {{moment-format (await obj.session.firstOfferingDate) "L"}}
+            {{#if obj.postrequisite}}
+              <b>
```

Comment:
My reading of https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#Usage_notes is that `strong` is preferred to `b`. What do you think?

Reply:
I was reading about it yesterday and the difference seems to be trivial. I am fine with `strong` as it signifies importance.